app/controllers/persons_controller.rb

```diff
@@ -18,84 +14,66 @@ def index
 
   def show
     @buildings = @person.spaces.map { |space| space.building }.uniq
-    @departments = @person.groups.sort.select { |group| group.group_type == "Department" }
+    @departments = @person.groups.select { |group| group.group_type == "Department" }.sort
```

Comment:
This is stylistic - Let's `.sor` on its own line so we ca more easily see the chaining.

Reply:
ok